inputs/ds_pipeline_part1_cw.json

```diff
@@ -0,0 +1 @@
+{"downSampling_01.bam_or_cram_file":"gs://fc-95ba75a4-38aa-4939-8e26-a617fb862a03/HG00405.final.cram","downSampling_01.bam_to_fq_disk_size":"${600}","downSampling_01.bam_to_fq_mem_size":"64 GB","downSampling_01.cram_to_bam_disk_size":"${200}","downSampling_01.cram_to_bam_mem_size":"15 GB","downSampling_01.downsample_docker":"gcr.io/neural-tangent-298217/downsample","downSampling_01.reference_fasta":"gs://fc-95ba75a4-38aa-4939-8e26-a617fb862a03/Homo_sapiens_assembly38.fasta"}
```

Comment:
This looks like the JSON downloaded from Terra, right? I imagine you've had to change it to run with cromwell, so I don't know what your current version looks like. But in case it'd be helpful, I just thought I'd direct you to [this resource](https://stackoverflow.com/questions/352098/how-can-i-pretty-print-json-in-a-shell-script) on how to make JSON formatting look nicer/more readable - my preferred method is ```python -m json.tool input.json > pretty.json```